src/main/java/com/matera/cadastrocentral/telephone/TelephoneController.java

```diff
@@ -58,11 +59,16 @@ public void deleteTelephone(@PathVariable("telephoneId") UUID telephoneId) {
         return telephoneService.findByClientIdAndTelephoneId(clientId, telephoneId);
     }
 
-    // 6. Update telephone in database
+    // 6. Update all telephone properties in database
     @PutMapping("/telephone")
-    public Telephone alterTelephoneByTelephoneId(@RequestBody @Validated TelephoneDTO telephone) {
+    public Telephone putTelephoneByTelephoneId(@RequestBody @Validated TelephoneDTO telephone) {
         return telephoneService.alterTelephoneByTelephoneId(telephone);
     }
 
+    // 7. Update telephone properties (one or more)
+    @PatchMapping("/telephones")
```

Comment:
Since we are adding a new endpoint, can we follow the pattern "/clients/{clientId}/telephones/{telephoneId}" here ?

Reply:
I considered in telephoneService that only "telephoneId" can't be changed, so I altered the PATCH request path to "/telephones/{telephoneId}", what do you think?

Reply:
gotcha, so we can change the client using the patch, right?

Reply:
Yes :)